mvpa2/cmdline/cmd_ttest.py

```diff
@@ -58,14 +61,38 @@ def setup_parser(parser):
                         belong to.  E.g. if values are accuracies, it would be the
                         'greater', if errors -- the 'less'""")
 
+def guess_backend(fn):
+    if fn.endswith('.gz'):
```

Comment:
would it solve any specific problem or make code shorter? ;)

what I am saying, that In general -- it might be worth it, but not sure if really necessary for the scope of this PR.